XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -249,6 +282,55 @@ private void Rebuild()
         }
 
         /// <summary>
+        /// Makes the warning that appears above the grid saying: "Automatic checking for updates 
+        /// is disabled for some types of updates" visible.
+        /// </summary>
+        private void MakeWarningVisible()
+        {
+            pictureBox1.Visible = true;
+            AutoCheckForUpdatesDisabledLabel.Visible = true;
+            checkForUpdatesNowButton2.Visible = true;
+        }
+
+        /// <summary>
+        /// Makes the warning that appears above the grid saying: "Automatic checking for updates 
+        /// is disabled for some types of updates" invisible.
+        /// </summary>
+        private void MakeWarningInvisible()
+        {
+            pictureBox1.Visible = false;
+            AutoCheckForUpdatesDisabledLabel.Visible = false;
+            checkForUpdatesNowButton2.Visible = false;
+        }
+
+        /// <summary>
+        /// Checks if the automatic checking for updates in the Updates Options Page is disabled for some, but not all types of updates.
+        /// </summary>
+        /// <returns></returns>
+        private bool SomeButNotAllUpdatesAllowed()

```

Comment:
I think this function could be renamed to SomeButNotAllUpdatesDisabled, as that's what is checking.
Similar for the SomeOrAllUpdatesAllowed below.